newIDE/app/src/MainFrame/EditorContainers/HomePage/PlaySection/GamesPlatformFrame.js

```diff
@@ -68,6 +71,24 @@ const GamesPlatformFrame = ({ initialGameId, loaded, visible }: Props) => {
     [loaded, initialGameId]
   );
 
+  React.useEffect(
```

Comment:
as discussed, this should probably be moved to the top component `GamesPlatformFrameContext` to do a logic similar to `sendUserCustomTokenToFrame()`